plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/price-slider/constants.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Order of colors in the array
+ * affects the order of CSS variables and classes
+ * in the generated CSS.
+ * Changing the order will break the markup
+ * generated by the block,
+ * producing validation errors.
```

Comment:
Nitpick: Can we use content width here, for example, 80 characters? The line breaks seem random to me.

Reply:
I do agree with 80 charts, but especially the maximum (vm ftw)
I tried to break the lines in the proper words, but let me improve it :-D.
Thanks for he feedback

Reply:
addressed here f9dd722ce5a531589804756912c489820cb37c4b